catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/NewProjectDialogFragment.java

```diff
@@ -120,12 +120,17 @@ void showOrientationDialog(String projectName, boolean createEmptyProject) {
 		dialog.show(getFragmentManager(), OrientationDialogFragment.TAG);
 	}
 
-	void createDroneProject(String name, boolean jumpingSumo) {
+	void createDroneProject(String name, boolean isJumpingSummoProject) {
 		try {
-			ProjectManager.getInstance().initializeNewProject(name, getActivity(), false, true,
-					false, false, jumpingSumo);
+			if (isJumpingSummoProject) {
```

Comment:
```suggestion
			if (isJumpingSumoProject) {
```
Please write "Sumo" only with one "m" everywhere in the project. This will make it easier at a later time to search and locate all places in the code related to the Sumo robot.